service/http/test_http_service.py

```
@@ -0,0 +1,51 @@
+import unittest
+from unittest.mock import patch, Mock
+from requests import Response
+from service.http.http_service import HttpService
+
+class TestHttpService(unittest.TestCase):
+
+    @patch('requests.get')
+    def test_get_success(self, mock_get):
```

Ruff

Missing return type annotation for public function `test_get_success`

Add return type annotation: `None`

(ANN201)